pkg/codegen/docs/gen.go

```diff
@@ -1957,8 +1957,14 @@ func (dctx *docGenContext) generatePackage(tool string, pkg *schema.Package) (ma
 
 	glog.V(3).Infoln("generating package docs now...")
 	files := fs{}
-	for _, mod := range dctx.modules() {
-		if err := mod.gen(files); err != nil {
+	modules := []string{}
```

Comment:
Oof nice find. Anything else using dctx.modules() like this?

Reply:
This is the only one